homeassistant/components/netgear/update.py

```diff
@@ -44,8 +45,7 @@ def __init__(
     ) -> None:
         """Initialize a Netgear device."""
         super().__init__(coordinator, router)
-        self._name = f"{router.device_name} Update"
```

Comment:
Only double-checking as the name will be changed to "Firmware". Is this fine?

Reply:
Yes that is fine, it is also the firmware of the router that will be updated.
It makes it more in line with the rest of HomeAssistant.